Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -615,6 +617,33 @@ - (nullable NSString *)currentScreenNameForSessionReplay
                .firstObject;
 }
 
+- (void)showMaskPreview:(CGFloat)opacity
+{
+    if ([SentryDependencyContainer.sharedInstance.crashWrapper isBeingTraced] == NO) {
+        return;
+    }
```

Comment:
`m`: What drove you to limit this to only when the debugger is attached?

Reply:
No releasing it by mistake